src/components/headerArea.scss

```diff
@@ -0,0 +1,160 @@
+.header-area {
+  padding: var(--contentPaddingPrimary);
+}
+
+.top {
+  display: flex;
+  border-top: 1px solid #000;
+  border-bottom: 1px solid #000;
+  padding: 24px 0 96px 0;
+}
+
+.top__title {
+  font-family: var(--visueltBold);
+  font-size: var(--headerFontSizePrimary);
+  line-height: 1;
+  color: var(--black);
+}
+
+.top__right-content {
+  font-family: var(--scopeOne);
+  font-size: var(--headerFontSizeTertiary);
+  font-weight: 400;
+  line-height: var(--headerLineHeightSecondary);
+  color: var(--black);
+  border-bottom: none;
+  margin-bottom: 0;
+}
+
+.top__left {
+  flex: 0 1 50%;
+  padding-right: 24px;
+}
+
+.top__right {
+  flex: 0 1 50%;
+  padding-left: 24px;
+}
+
+.bottom__label {
+  display: inline-block;
+  font-family: var(--ibmPlexMono);
+  font-size: var(--linkFontSize);
+  letter-spacing: 1px;
+  text-transform: uppercase;
+  margin-bottom: 16px;
+  color: var(--black);
+}
+
+.rich-text {
+  font-family: var(--visueltRegular);
+  font-size: var(--paragraphFontSize);
+  line-height: var(--paragraphLineHeight);
+  margin-bottom: 28px;
+  color: var(--black);
+}
+
+// about page specific styles
```

Comment:
I think it probably makes sense to move these specific page styles out of the headerArea component and into their own components or into the specific page component itself.

I'm open to other ways of handling this, but I can see us ending up with a really long list of styles here. It seems like we have 3 or 4 types of content being displayed in the bottom portion of the header, so perhaps we can make those components themselves and just pull them in as we need.

That being said, I think it is time to hook up Contentful before we make any changes so we can see how that data comes in and any potential complications.